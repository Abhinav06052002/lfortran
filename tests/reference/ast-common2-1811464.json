{
    "basename": "ast-common2-1811464",
    "cmd": "lfortran --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/common2.f",
    "infile_hash": "761ba9bcb69efc19ad1828c9ebc8deab710d02f98f6722736d6a4dd2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-common2-1811464.stdout",
    "stdout_hash": "c924dca05c0f9bf92e16edc12f752e6a4968695aeaa179aaab6f0cf6",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}