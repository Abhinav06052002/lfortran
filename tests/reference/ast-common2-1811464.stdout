(TranslationUnit
    [(Subroutine
        f
        []
        []
        ()
        ()
        []
        []
        []
        [(Declaration
            (AttrType
                TypeInteger
                []
                ()
                ()
                None
            )
            []
            [(a
            []
            []
            ()
            ()
            None
            ())
            (b
            []
            []
            ()
            ()
            None
            ())
            (c
            []
            []
            ()
            ()
            None
            ())
            (d
            []
            []
            ()
            ()
            None
            ())
            (e
            []
            []
            ()
            ()
            None
            ())
            (g
            []
            []
            ()
            ()
            None
            ())
            (h
            []
            []
            ()
            ()
            None
            ())
            (i
            []
            []
            ()
            ()
            None
            ())
            (j
            []
            []
            ()
            ()
            None
            ())
            (k
            []
            []
            ()
            ()
            None
            ())]
            ()
        )
        (Declaration
            ()
            [(SimpleAttribute
                AttrCommon
            )]
            [(block_1
            []
            []
            ()
            ()
            Slash
            ())
            (a
            []
            []
            ()
            a
            None
            ())
            (b
            []
            []
            ()
            b
            None
            ())
            (h
            []
            []
            ()
            h
            None
            ())
            (block_2
            []
            []
            ()
            ()
            Slash
            ())
            (c
            []
            []
            ()
            c
            None
            ())
            (d
            []
            []
            ()
            d
            None
            ())
            (block_3
            []
            []
            ()
            ()
            Slash
            ())
            (e
            []
            []
            ()
            e
            None
            ())
            (g
            []
            []
            ()
            g
            None
            ())]
            ()
        )
        (Declaration
            ()
            [(SimpleAttribute
                AttrCommon
            )]
            [(block_4
            []
            []
            ()
            ()
            Slash
            ())
            (i
            []
            []
            ()
            i
            None
            ())
            (block_5
            []
            []
            ()
            ()
            Slash
            ())
            (j
            []
            []
            ()
            j
            None
            ())
            (k
            []
            []
            ()
            k
            None
            ())]
            ()
        )]
        []
        []
        []
    )]
)
